internal/encoding/block/from_parquet.go

```diff
@@ -92,3 +99,49 @@ func FromParquetBy(payload []byte, partitionBy string, filter *typeof.Schema, ap
 	blocks = append(blocks, last...)
 	return blocks, nil
 }
+
+type fieldHandler func(interface{}) (interface{}, error)
+
+func getReverseHandler(typ string) (fieldHandler func(interface{}) (interface{}, error), err error) {
+
+	switch typ {
+	case "string":
+		fieldHandler = stringHandler
+	}
+
+	return fieldHandler, nil
+}
+
+// Transform runs the computed Values and overwrites/appends them to the set.
+func ApplyCastHandlers(r Row, fieldHandlers []fieldHandler) Row {
+	// Create a new output row and copy the column values from the input
+	schema := make(typeof.Schema, len(r.Schema))
+	out := NewRow(schema, len(r.Values))
+
+	i := 0
+	for k, v := range r.Values {
+		handler := fieldHandlers[i]
+
+		if handler != nil {
+			out.Values[k], _ = handler(v)
+		}
+		out.Schema[k] = r.Schema[k]
+	}
+
+	return out
+}
+
+func stringHandler(s interface{}) (interface{}, error) {
```

Comment:
Change the name to something like `parquetString` since we're in a `block` package